Etabs_Adapter/Helpers/Results.cs

```diff
@@ -309,6 +329,97 @@ public static List<BarForce> GetBarForce(cSapModel model, IList ids = null, ILis
             return barForces;
         }
 
+
+        public static List<PierForce> GetPierForce(cSapModel model, IList ids = null, IList cases = null, int divisions = 5)
+        {
+
+            List<string> loadcaseIds = new List<string>();
+            List<string> barIds = new List<string>();
+            List<PierForce> pierForces = new List<PierForce>();
+
+            if (cases == null)
```

Comment:
Would consider using the `CheckAndGetCases` method here instead!

Reply:
done